data/data/bootstrap/files/usr/local/bin/installer-control-plane-gather.sh

```diff
@@ -0,0 +1,55 @@
+# Collect cluster data
```

Comment:
```suggestion
#!/usr/bin/env bash

# Collect cluster data
```

Reply:
It's meant to be sourced and not ran directly but I guess it won't harm